sim/druid/swipe.go

```diff
@@ -62,18 +50,31 @@ func (druid *Druid) registerSwipeBearSpell() {
 			IgnoreHaste: true,
 		},
 
-		DamageMultiplier: 1 + 0.1*float64(druid.Talents.SavageFury),
+		DamageMultiplier: baseMultiplier + 0.1*float64(druid.Talents.SavageFury),
 		ThreatMultiplier: SwipeThreatMultiplier,
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
 			for idx := range results {
+				dotBonusCrit := 0.0
+				if druid.LacerateBleed.Dot(target).GetStacks() > 0 {
```

Comment:
Will this potentially error if you don't have the Lacerate rune equipped? You might need to check if they have the lacerate rune (another cached call)